pyscriptjs/Makefile

```diff
@@ -6,14 +6,15 @@ src_dir ?= $(base_dir)/src
 examples ?=  $(base_dir)/examples
 app_dir ?= $(shell git rev-parse --show-prefix)
 
+CONDA_EXE := conda
```

Comment:
@jezdez  do you have any opinions about this? Is that fine from a pure conda perspective?

Reply:
For context, it's basically just conda from PATH as the setup-conda action allows, but this variable existed in the makefile previously and was undefined.